engine/previews/citizens_advice_components/table_preview/long_table.html.haml

```diff
@@ -0,0 +1,6 @@
+.cads-page-content
```

Comment:
Does this need all the layout classes? Guessing this is what the previous example did?

Reply:
There was no example outside of the sample pages so I just added some in to illustrate the scrolling behaviour.  Probably don't need _all_ of them...